framework/CodeInterfaces/CobraTF/ctfdata.py

```diff
@@ -0,0 +1,551 @@
+                        key_name = 'fuel_rod' + str(fuel_no)
+
+                        # line filtering due to the existence of '*' in the output
+                        if '*' in line:
+                            line_filtered = line.replace('*', "")
+                        else:
+                            line_filtered = line
+
+                        # print(line_filtered)
+                        for j in range(2, len(line_filtered.split())):
+                            var_no += 1
+                            # skip the non-numeric elements
+                            if j != 5:
+                                DictArray[timestep_cnt - 1,
+                                          var_no] = line_filtered.split()[j]
+
+                            if header_w:
+                                # print(header_w)
+                                if j == 2:
+                                    header_name = key_name + '_fluidTemperatureLiquid'
+                                    header.append(header_name)
+                                elif j == 3:
+                                    header_name = key_name + '_fluidTemperatureVapor'
+                                    header.append(header_name)
+                                elif j == 4:
+                                    header_name = key_name + '_surfaceHeatflux'
+                                    header.append(header_name)
+                                elif j == 5:
+                                    header_name = key_name + '_heatTransferMode'
+                                    header.append(header_name)
+                                elif j == 6:
+                                    header_name = key_name + '_caldOutTemperature'
+                                    header.append(header_name)
+                                elif j == 7:
+                                    header_name = key_name + '_cladInTemperature'
+                                    header.append(header_name)
+                                elif j == 8:
+                                    header_name = key_name + '_gapConductance'
+                                    header.append(header_name)
+                                elif j == 9:
+                                    header_name = key_name + '_fuelTemperatureSurface'
+                                    header.append(header_name)
+                                elif j == 10:
+                                    header_name = key_name + '_fuelTemperatureCenter'
+                                    header.append(header_name)
+                                else:
+                                    raise IOError(
+                                        "Error: Unexpected output file format. Check the oufput file (output type 4, fuel rod).")
+
+                        # at end of rod info, reset o_type3 booleans
+                        if fuel_no == 1:
+                            o_type4 = False
+
+        return DictArray, header
```

Comment:
General coment on this file... None of the variables are ```camelBack```. Please fix all of them